### DISCUSSION TITLE
VRoid SDK0.1.0 プレビュー版リリース告知

### DISCUSSION THREAD

**Discussion by Haresaku** [pixiv/vroid-sdk-developers#23] [Announcements]
VRoid SDK0.1.0 プレビュー版を12月中旬にリリースする予定です！

既知の問題への対応、体験の改善を含むアップデートとなります。いくつかのBreaking Changeを含むため、まずはプレビュー版を開発者のみなさまに試していただき、フィードバックをいただけますと幸いです。

- 認証コードのコピーなしに、ブラウザの連携ボタンをクリックすることでアプリケーション側との連携を行えるように
- アプリケーション設定をcredential.jsonとして、アプリケーション設定画面から一括でダウンロードし、アプリケーションに簡単に反映できるように
- 同梱しているUniVRMのアップデート
- 開発環境のUnity2019へのアップデート
- 既知の問題への対応
    - Windows環境で3Dモデル取得時にモデルを展開できないバグ対策を導入
        - 通信方式にUnityWebRequestから.NET標準のHttpClientを利用した通信方式を提供
        - 加えて自前で仕組みをカスタマイズできるようにインターフェースを提供
    - iOSでの認証時にURLスキーマ周りで発生するエラーの解決
        - Spectrumに記載のワークアラウンドを導入

今後ともVRoid SDKの定期的なアップデートを予定しております。

引き続き、VRoid SDKをよろしくお願いいたします。

**Comment by elvneko** [comment]
楽しみにしております！
12月中旬とのことですが既に利用可能でしょうか？

**Reply by Haresaku** [reply]
現在リリース準備中です！もうしばらくお待ちを
